keyboards/ergodash/keymaps/default/keymap.c

```diff
@@ -0,0 +1,137 @@
+     F11, F1,  F2,  F3,  F4,  F5, LCBR,          RCBR, F6,  F7,  F8,  F9,  F10, F12,
+ //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+     TILD,EXLM, AT ,HASH, DLR,PERC,UNDS,         PLUS,CIRC,AMPR,ASTR,LPRN,RPRN,PIPE,
+ //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+     TAB , 1  , 2  , 3  , 4  , 5  ,DEL ,         BSPC,  H  , J ,  K ,  L ,COLN, DQT,
+ //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+     LSFT, 6  , 7  , 8  , 9  , 0  ,JPN,           ENT,  N  , M , LT , GT ,QUES,RSFT,
+ //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+    LCTL,LGUI,LALT, AJST,     LOWR,SPC,           ENT,RASE,     HOME,PGDN,PGUP, END
+ //`----+----+----+----+----+----+----'         `----+----+----+----+----+----+----'
+ ),
+
+/* Raise
+ */
+  [_RAISE] = KC_KEYMAP(
+  //,----+----+----+----+----+----+----.         ,----+----+----+----+----+----+----.
+      F11, F1,  F2,  F3,  F4,  F5, LCBR,          RCBR, F6,  F7,  F8,  F9,  F10, F12,
+  //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+      TILD,EXLM, AT ,HASH, DLR,PERC,UNDS,         PLUS,CIRC,AMPR,ASTR,LPRN,RPRN,PIPE,
+  //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+      TAB , 1  , 2  , 3  , 4  , 5  ,DEL ,         BSPC,  H  , J ,  K ,  L ,COLN, DQT,
+  //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+      LSFT, 6  , 7  , 8  , 9  , 0  ,JPN,           ENT,  N  , M , LT , GT ,QUES,RSFT,
+  //|----+----+----+----+----+----+----|         |----+----+----+----+----+----+----|
+     LCTL,LGUI,LALT, AJST,     LOWR,SPC,           ENT,RASE,     HOME,PGDN,PGUP, END
+  //`----+----+----+----+----+----+----'         `----+----+----+----+----+----+----'
+  ),
+
+  [_ADJUST] = KEYMAP(
+    _______, _______, _______, _______, _______, _______,_______,       _______, _______, _______, _______, _______, _______, _______, \
+    _______, RESET  , RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI,_______,       _______, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, _______, _______, \
+    _______, _______, _______, _______, _______, _______,_______,       _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,_______,       _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, _______, _______,          _______,_______,       _______, _______,          _______, _______, _______, _______ \
+  )
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
```

Comment:
The `persistent_default_layer_set` function, and the manual audio tones for it have been depreciated in favor of `set_single_persistent_default_layer`, which handles both: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L190

To define the audio, you just use this:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/config.h#L10